browser/brave_profile_prefs.cc

```diff
@@ -199,6 +199,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   // Default Brave shields
   registry->RegisterBooleanPref(kHTTPSEVerywhereControlType, true);
   registry->RegisterBooleanPref(kNoScriptControlType, false);
+  registry->RegisterBooleanPref(kReduceLanguageEnabled, true);
```

Comment:
BraveFarblingService::RegiserProfilePrefs

Reply:
Fixed in https://github.com/brave/brave-core/pull/12234/commits/e937302cb6925c96fb2ea026788eb4409d2e325c